bin/imsim.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```

Comment:
On Mac this is causing the program to fail.  I believe this is do to SIP stripping the LIBRARY_PATH from the sub process.  We went through a lot of this before.  @timj can probably tell us the "approved" DM solution.  In the mean time running this as "python bin/imsim" works.